pkg/debug/scripts/collect-info.sh

```diff
@@ -15,19 +15,26 @@ VERSION=7
 # still attempt to install those packages.
 PKG_DEPS="procps tar dmidecode iptables dhcpcd"
 
-DATE=$(date -Is)
+DATE=$(date "+%a-%d-%b-%Y-%H-%M-%S")
```

Comment:
I have no issue with getting rid of the colons, but can we use a format which sorts properly (year, month, date) such as date "+%Y-%m-%d-%H-%M-%S"

Reply:
Of course